pkgs/tools/misc/nautilus-open-any-terminal/default.nix

```diff
@@ -52,7 +52,7 @@ python3.pkgs.buildPythonPackage rec {
     glib-compile-schemas "$out/share/glib-2.0/schemas"
   '';
 
-  PKG_CONFIG_LIBNAUTILUS_EXTENSION_EXTENSIONDIR = "${placeholder "out"}/lib/nautilus/extensions-3.0";
+  PKG_CONFIG_LIBNAUTILUS_EXTENSION_EXTENSIONDIR = "${placeholder "out"}/lib/nautilus/extensions-4";
```

Comment:
The variable would be `PKG_CONFIG_LIBNAUTILUS_EXTENSION_4_EXTENSIONDIR`. But It does not look to me like this is used anywhere – this is an extension of nautilus-python, not a Nautilus one.

Reply:
I just noticed the miss config in my system. It was issing:
`environment.sessionVariables.NAUTILUS_4_EXTENSION_DIR = "${config.system.path}/lib/nautilus/extensions-4";`
Now it works even without this change. So we might close it?

Reply:
Ideally, we would remove this line completely.

Reply:
sure. are there plans to provide something like `nautilus-with-extensions`? This would get rid of all these link problems.

Reply:
It is on my to do list but very low priority. So it probably will not happen until someone else opens a PR.